src/filesystem/README.md

```diff
@@ -82,10 +82,12 @@ Node.js server implementing Model Context Protocol (MCP) for filesystem operatio
 Add this to your `claude_desktop_config.json`:
 ```json
 {
-  "filesystem": {
-    "command": "npx",
-    "args": ["-y", "@modelcontextprotocol/server-filesystem", "/Users/username/Desktop", "/path/to/other/allowed/dir"]
-  }
+    "mcpServers": {
+      "filesystem": {
+        "command": "npx",
+        "args": ["-y", "@modelcontextprotocol/server-filesystem", "/Users/username/Desktop", "/path/to/other/allowed/dir"]
+      }
+    }
```

Comment:
Really minor nit: can you please make sure to use 2-space indents in the excerpt here? (There are 4 spaces for the outermost indent level right now.)